docs/platform/howto/access-service-logs.rst

```diff
@@ -15,8 +15,8 @@ There are three built-in ways to inspect service logs at Aiven:
 
 * `Aiven API <https://api.aiven.io/doc/#operation/ProjectGetServiceLogs>`_ endpoint is available for fetching the same information two above methods output, in case programmatic access is needed.
 
-.. Warning::
-
-    Service logs are stored only for a few days. Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
+Log Retention Policy
+--------------------
+Service logs are generally retained for a few days, and in some cases up to 14 days maximum.  Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
```

Comment:
@bcrisologo I would mention Aiven Services instead "some cases"

Service logs are generally retained for a few days, and for some Aiven Services up to 14 days maximum.  Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.

Reply:
Updated pull request to apply the change above